src/UnicycleNavigationController.cpp

```diff
@@ -120,14 +121,19 @@ bool UnicycleNavigationController::computeDesiredVelocities()
     }
 
     double angleDifference = m_navigationPath[m_poseIndex].angle - m_state.angle;
-    //ETAs computation
-    double linearETA = distance / linearSpeed;  //Time required for moving from (x_i, y_i) to (x_i+1, y_i+1)
-    double absoluteAngleDifference = std::abs(angleDifference); 
-    if (absoluteAngleDifference >= M_PI)
+
+    if (angleDifference > M_PI)
     {
-        absoluteAngleDifference = std::abs(absoluteAngleDifference - 2*M_PI);
+        angleDifference = angleDifference - 2*M_PI;
     }
-    double angularETA = absoluteAngleDifference / m_maxAngularVelocity;   //Time required for moving from theta_i to theta_i+1
+    else if (angleDifference < -M_PI)
+    {
+        angleDifference = angleDifference + 2*M_PI;
+    }
+
+    //ETAs computation
+    double linearETA = distance / linearSpeed;  //Time required for moving from (x_i, y_i) to (x_i+1, y_i+1)
```

Comment:
This can trigger a division by zero. Can you handle that case?

Reply:
`linearSpeed` is always > 0 since it's computed by [this statement](https://github.com/SimoneMic/unicycle-footstep-planner/blob/bbb3886a867d94a852f4eff2f4f15e5e2fedc17c/src/UnicycleNavigationController.cpp#L100)
And both `m_maxLateralVelocity` and `m_maxVelocity` are > 0 by design

Reply:
Ok, great!